Digicert ca decommissioning (#447)
GSA/fpki-guides: _announcements/01_fpki-repository-migration.md

pubDate: April 1, 2019
collection: announcements
permalink: announcements/fpki-repository-migration/
description: On April 22, 2019, the Federal Public Key Infrastructure Management Authority will migrate the hosting of HyperText Transfer Protocol (HTTP) repository services to a cloud-based solution. This announcement provides additional information related to the upcoming migration.
---

{% include alert-info.html content="Upcoming changes to the hosting of Federal Public Key Infrastructure Certification Authority (CA) data repositories could impact your agency." %}

On April 22, 2019, the Federal Public Key Infrastructure Management Authority will migrate the hosting of HyperText Transfer Protocol (HTTP) repository services to a cloud-based solution. Existing Federal PKI CA certificate Uniform Resource Locators (URLs) **will not** change as a result of this migration.

## What will be impacted?

This change will affect the hosting of certificate revocation lists, CA certificates, and certificate bundles for the following Federal PKI CAs:
- Federal Bridge CA 2016
- Federal Common Policy CA
- SHA1 Federal Root CA

> - CAs operating within the FPKI Community Interoperability Test Environment (CITE)

Reviewer: ```suggestion
- Some Test CAs operating for the FPKI Community Interoperability Test Environment (CITE)
```

> - US Federal TLS Root CA (under development) 

Reviewer: ```suggestion
```

Reviewer: If it's the Dev / Test CA, let's be specific. 
The production CA doesn't exist yet so we don't need to specify this in the announcement (reduce confusion).

## When will this change take place?
The migration will take place on April 22, 2019.

## What should I do?

> This change will be transparent to Relying Parties, and should not require any agency action. 

Reviewer: @ryancdickson let's also update the piv guides.

The FPKI Community Interoperability Test Environment HTTP repository (http://http.cite.fpki-lab.gov) has used the new service since June 2018 with no reported issues. 

A new base URL is available for anyone who would like to test the planned repository service update before the April 22, 2019 migration. For example, to download a copy of the Federal Common Policy CA certificate using the cloud-based hosting solution, navigate to http://cdn.http.fpki.gov/fcpca/fcpca.crt. 

Contact fpki-help@gsa.gov with the subject “CDN Test Issue” if you'd like to learn more about testing, or if you have any issues.

## Who can I contact for help or more information?

> Email us at fpki@gsa.gov. 

Reviewer: ```suggestion
Email us at fpki-help@gsa.gov. 
```
